capstone/src/main/webapp/script.js

```diff
@@ -277,24 +277,13 @@ function loadResponseChart(totalResponses, precinct) {
   chart.draw(stats, null);
 }
 
-function mapSentiment(colorMap) {
-  const precinctLayer = mapAndSelection.map;
-  for (const [precinctName, precinctColor] of colorMap) {
-    precinctLayer.revertStyle();
-    precinctLayer.setStyle({fillColor: '#CECDBC',
-      fillOpacity: 0.9});
-    precinctLayer.overrideStyle(precinctLayer.getFeatureById(
-        searchPrecinctsByStation(precinctName)),
-    {fillColor: precinctColor, fillOpacity: 0.9});
-  }
-}
-
 function searchPrecinctsByStation(desiredStation) {
-  mapAndSelection.map.forEachFeature( function(feature) {
-    if (feature.getProperty('station') == desiredStation) {
-      return feature.getProperty('id');
+    let precinctID = [200,300,400,500,600,800,900,1000];
```

Comment:
Are these randomly assigned values for the precincts?

Reply:
IDs are numbers, there is a station property for each precinct feature though.

Reply:
Why are the ids 200-1000?

Reply:
because there are 10 precincts, I added two 0s after each number and I haven't drawn precinct 1 or 7.

Reply:
A comment explaining that would be good 👍